index.js

```diff
@@ -81,15 +83,27 @@ function Server () {
     const db = require('./server/models');
     const PORT = siteConfig.details.port;
     // sync sequelize
-    db.sequelize.sync()
-    // start the server
+    createDatabaseIfNotExists()
+      getWalletBalance()
+      .then(balance => {
+        logger.info('starting LBC balance:', balance);
+      })
+      .then(() => {
+        db.sequelize.sync()
+      })
```

Comment:
These can all live inside one `.then` after the `getWalletBalance()`

No need to break them up

Reply:
I tried updating a bit, is this what you had in mind?  I'm still a little fuzzy on the best patterns for chaining and when to use `.then`, but I suppose `db.sequelize.sync` and `this.server.listen` can be handled asynchronously